Mathlib/MeasureTheory/Group/FundamentalDomain.lean

```diff
@@ -718,4 +727,289 @@ protected theorem fundamentalInterior : IsFundamentalDomain G (fundamentalInteri
 
 end IsFundamentalDomain
 
+section FundamentalDomainMeasure
+
+variable (G) [Group G] [MulAction G α] [MeasurableSpace α]
+  (μ : Measure α)
+
+local notation "α_mod_G" => MulAction.orbitRel G α
+
+local notation "π" => @Quotient.mk _ α_mod_G
+
+variable {G}
+
+@[to_additive addQuotientMeasure_apply]
+lemma measureRestrictMap_apply (s : Set α) {U : Set (Quotient α_mod_G)} (meas_U : MeasurableSet U) :
+    (μ.restrict s).map π U = μ ((π ⁻¹' U) ∩ s) := by
+  rw [map_apply (f := π) (fun V hV ↦ measurableSet_quotient.mp hV) meas_U,
+    Measure.restrict_apply (t := (Quotient.mk α_mod_G ⁻¹' U)) (measurableSet_quotient.mp meas_U)]
+
+@[to_additive IsAddFundamentalDomain.addQuotientMeasure_eq]
```

Comment:
You shouldn't need to provide the additive names by yourself: in the file `Mathlib.Tactic.ToAdditive`, you can register the fact that `IsFundamentalDomain` should be additivized to `IsAddFundamentalDomain`, once and for all, as well as all the other terms you use.